xml/admin_administration.xml

```diff
@@ -205,40 +205,45 @@
   <sect2 xml:id="commands.node.managment.issue-generator">
    <title>The <command>issue-generator</command> Command</title>
    <para>
-    The <command>issue-generator</command> creates a volatile temporary
-    <filename>/run/issue</filename> file. The file
-    <filename>/etc/issue</filename> should be a symbolic link to the temporary
-    <filename>/run/issue</filename>.
+    On &suse; &mos;, <filename>/etc/issue</filename> is generated by <command>issue-generator</command>.
+   </para>
```

Comment:
Do we really need a new para after just once sentence?

Reply:
Done.